dags/example_dynamodb.py

```diff
@@ -0,0 +1,110 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Other than the one addition below, this file should live in `tests/system/providers/amazon/aws/example_dynamodb.py`

Reply:
And you'll need to add the START and END tags for the code snippet(s) for docs page, but I presume that is Coming Soon :tm:.

Reply:
Done.